Automattic/wp-calypso
# Promotions: Reset current page upon search

This moves the search filter into the redux store where a reducer can
both set the filter and reset the page number to 1 whenever a new search
is entered.

 To Test:
1. `npm test`
2. `npm start`
3. Go to promotions page.
4. Flip through pages of promotions to ensure proper behavior.
5. From page 2 or above, type in search term to narrow results.
6. Observe that page 1 is active now.
